### ISSUE TITLE
Accessibility of installation page

### ISSUE THREAD

**Issue by steverep** [home-assistant/analytics.home-assistant.io#910]
The charts on the installation page are completely inaccessible to screen readers and other assistive technology.  By far the best way to resolve this is to offer the information in a tabular format as well.  This should be particularly easy for the pie charts.
